spec-0007/index.md

```diff
@@ -5,69 +5,69 @@ author:
   - "Stéfan van der Walt <stefanv@berkeley.edu>"
   - "Sebastian Berg <sebastianb@nvidia.com>"
   - "Pamphile Roy <roy.pamphile@gmail.com>"
+  - "Matt Haberland <mhaberla@calpoly.edu>"
   - Other participants in the discussion <not.yet@named.org>"
 discussion: https://github.com/scipy/scipy/issues/14322
 endorsed-by:
 ---
 
 ## Description
 
-<!--
-Briefly and clearly describe the proposal.
-Explain the general need and the advantages of this specific proposal.
-If relevant, include examples of how the new functionality would be used,
-intended use-cases, and pseudo-code illustrating its use.
--->
-
-There is disparity in the APIs libraries provide to seed random number generation.
-This SPEC suggests a single, pragmatic API for the ecosystem, taking into account technical and historical factors.
+Currently, libraries across the ecosystem provide various APIs for seeding random number generation.
+This SPEC suggests a unified, pragmatic API, taking into account technical and historical factors.
 Adopting such a uniform API will simplify the user experience, especially for those who rely on multiple projects.
 
-We recommend to:
+We recommend:
+
+- avoiding the use of global state and legacy bitstream generators, and
+- standardizing the usage and interpretation of an `rng` keyword for seeding.
+
+We suggest implementing these principles by:
 
-- Standardize the usage and interpretation of an `rng` keyword for seeding, and
-- avoid the use of global state and legacy bitstream generators.
+- deprecating the use of `numpy.random.seed` to control the random state,
```

Comment:
This said something before about deprecating use of `RandomState`s, but that is not really done explicilty. It is implicit: use of old `random_state`/`seed` arguments are deprecated, and the `rng` argument is validated by `default_rng`, so `RandomState`s won't be accepted.

Reply:
It says it above now, "legacy bitstream generators". We could, in brackets, add `RandomState` there.